### PR TITLE
ISSD-1821 Create the Release Notes page - 3rd tab: Breaking Changes

### PR DESCRIPTION
**Pull request by samuelleolopes**
[LRSD-1821](https://liferay.atlassian.net/browse/LRSD-1821)

### CODE REVIEW

**Review by amosfong — APPROVED** [review]
Looks good :)

**Comment by amosfong** [comment]
@ryanschuhler , can you review, thanks!

**Review by ryanschuhler — CHANGES_REQUESTED** [review]
@samuelleolopes can you apply the same changes discuss in the prior pull to this one as well? (style namespacing, h1 usage, rest variable usage) and resend. Other than that looks good!

**Comment by samuelleolopes** [comment]
Thanks! @ryanschuhler  and @amosfong, adjustments made.

**Comment by amosfong** [comment]
@ryanschuhler , can you review, thanks!

**Comment by ryanschuhler** [comment]
ci:test:sf

**Review by ryanschuhler — APPROVED** [review]
Looks good, thanks!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+	}
+
+	.collapse-container {
+		position: relative;
+	}
+
+	.collapse-container .tooltip {
+		margin-top: -100px;
+		transform: translateX(-40%);
+		width: 140px;
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl]: Can't it be `rem` here? To be consistent with the rest of the file

**Review by pedroliveirav — CHANGES_REQUESTED** [review]
Very good Samuel, just one small change 🔨

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+		line-height: 1.25rem;
+	}
+
+	.empty-result-message {
+		align-items: center;
+		margin: 5rem auto;
+	}
+
+	.icon-container {
+		background-color: #E7EFFF;
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl]: Isn't there a corresponding color in frontend-tokens-values.json?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+		border: 0.0625rem solid var(--color-brand-primary-lighten-5, #E7EFFF);
+		border-radius: 0.625rem;
+		color: var(--color-action-neutral-default, #2B3A4B);
+		font-size: 0.875rem;
+		padding: 0.75rem;
+		word-break: break-word;
+	}
+
+	.link-container>svg {
+		fill: #0B5FFF;
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl]: Isn't there a corresponding color in frontend-tokens-values.json?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+		font-size: 0.8125rem;
+		font-style: normal;
+		font-weight: 400;
+		gap: 0.25rem;
+		line-height: 1rem;
+		margin-top: 0.5rem;
+	}
+
+	.search-icon>svg {
+		fill: #0B5FFF;
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl]: Isn't there a corresponding color in frontend-tokens-values.json?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl (hunk 1 above)
Comment (samuelleolopes) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl]: The class was removed and not in use.

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
Thanks @pedroliveirav !

**Review by pedroliveirav — APPROVED** [review]
Good job!

**Comment by samuelleolopes** [comment]
Waiting for the Tab 2 PR to be merged first.

**Review by kylembischof — APPROVED** [review]

**Comment by samuelleolopes** [comment]
ci:forward:force

**Pull request closed without merging**